frontend/src/css/App.css

```diff
@@ -15,6 +15,22 @@ Formatting for English/French button
   top: 3.5%;
 }
 
+/*
+Formatting for Back to top button; based off of Aurora
+*/
+.btn-back-to-top {
```

Comment:
If this is based on Aurora, can we use the same className: `.btn-backtotop`
And then just update the colours of `.btn-backtotop` that we want in App.css. The rest of the CSS can stay the same and be pulled from Aurora.

Reply:
I didn't know I could do that! Very useful. Thanks.
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/87/commits/b9a8652e6dac81dd546d50365f5f5decb12fe870